testplans/lotus-soup/go.mod

```diff
@@ -8,30 +8,30 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/drand/drand v1.2.1
 	github.com/filecoin-project/go-address v0.0.5
-	github.com/filecoin-project/go-data-transfer v1.7.8
-	github.com/filecoin-project/go-fil-markets v1.8.1
+	github.com/filecoin-project/go-data-transfer v1.10.1
+	github.com/filecoin-project/go-fil-markets v1.12.0
 	github.com/filecoin-project/go-jsonrpc v0.1.4-0.20210217175800-45ea43ac2bec
 	github.com/filecoin-project/go-state-types v0.1.1-0.20210810190654-139e0e79e69e
 	github.com/filecoin-project/go-storedcounter v0.0.0-20200421200003-1c99c62e8a5b
 	github.com/filecoin-project/lotus v0.0.0-00010101000000-000000000000
 	github.com/filecoin-project/specs-actors v0.9.14
-	github.com/google/uuid v1.2.0
-	github.com/gorilla/mux v1.7.4
-	github.com/hashicorp/go-multierror v1.1.0
+	github.com/google/uuid v1.3.0
+	github.com/gorilla/mux v1.8.0
+	github.com/hashicorp/go-multierror v1.1.1
+	github.com/influxdata/influxdb v1.9.4 // indirect
 	github.com/ipfs/go-cid v0.1.0
-	github.com/ipfs/go-datastore v0.4.5
+	github.com/ipfs/go-datastore v0.4.6
 	github.com/ipfs/go-ipfs-files v0.0.8
 	github.com/ipfs/go-ipld-format v0.2.0
 	github.com/ipfs/go-log/v2 v2.3.0
 	github.com/ipfs/go-merkledag v0.3.2
 	github.com/ipfs/go-unixfs v0.2.6
-	github.com/ipld/go-car v0.1.1-0.20201119040415-11b6074b6d4d
+	github.com/ipld/go-car v0.3.1-0.20210601190600-f512dac51e8e
```

Comment:
This is not a commit on master, or a tagged commit.

Reply:
@Stebalien This has been a problem with the `go-car` dep for a while now. Do you just want us to tag this commit and push it in ?

Reply:
That works. As long as we can be sure it won't disappear.

Reply:
@Stebalien - This is done. Please can you approve ?